priam/src/main/java/com/netflix/priam/tuner/StandardTuner.java

```diff
@@ -1,53 +1,49 @@
 /**
```

Comment:
Can you avoid clubbing formatting changes with the code changes? This helps in quality code review.